### PR TITLE
Add progress indicators tab to view status during event

### PR DESCRIPTION
**Pull request by martijnjanssen**
image
This page takes properties of checked in users and consumptions directly from the api and shows the progress so far.

### CODE REVIEW

File: src/lancie-admin-progress/lancie-admin-progress-meter.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/paper-progress/paper-progress.html">
+
+<dom-module id="lancie-admin-progress-meter">
```
Comment (elarb) [inline src/lancie-admin-progress/lancie-admin-progress-meter.html:4]: Is a new element needed?

File: src/lancie-admin-progress/lancie-admin-progress.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
```
Comment (elarb) [inline src/lancie-admin-progress/lancie-admin-progress.html:1]: Just `polymer/polymer-element.html` is sufficient

File: src/lancie-admin-progress/lancie-admin-progress.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      request.response.forEach(consumption => {
+        this.consumptions[consumption.name] = 0;
+      });
+      this.$.ajaxConsumptionMaps.generateRequest();
+    }
+  }
+
+  // Increases bucket for each consumption for each time it is consumed
+  onConsumptionMaps(event, request) {
+    if (request.succeeded) {
```
Comment (elarb) [inline src/lancie-admin-progress/lancie-admin-progress.html:73]: Maybe 
```
if (!request.succeeded) {
return;
}
...
```

would be better?

**Review by elarb — COMMENTED** [review]
Looks good!

File: src/lancie-admin-progress/lancie-admin-progress-meter.html (hunk 1 above)
Comment (martijnjanssen) [reply src/lancie-admin-progress/lancie-admin-progress-meter.html:4]: I wasn’t sure what I was aiming for, but if we want to expand it, we can. I can just change it and put it in the parent. What does the rest think about this?

File: src/lancie-admin-progress/lancie-admin-progress.html (hunk 3 above)
Comment (martijnjanssen) [reply src/lancie-admin-progress/lancie-admin-progress.html:73]: Yeah, I agree. Maybe this is a good formatting to use for future on-response functions.

**Review by elarb — APPROVED** [review]

**Pull request merged**